src/components/Badge/Badge.tsx

```diff
@@ -28,17 +28,18 @@ const Badge = React.forwardRef<HTMLElement, BadgeProps>(function Badge(
       aria-atomic="true"
       ref={ref}
       cursor="default"
-      width="fit-content"
-      minWidth="85px"
+      width={size === 'medium' ? 'fit-content' : 'auto'}
```

Comment:
hmmm if it's fit-content and `auto` then how is this `medium` or `large`? Can you explain this change a bit more?